src/Pages/ClientsPage/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react';
+import { DataTable, Pagination } from '../../Components/DataTable';
+import Navbar from '../../Components/Navbar';
+import Search from '../../Components/Search';
+import { GetClients } from '../../Services/ClientsService';
+
+function ClientsPage() {
+  const defaultPageSize = 8;
+  const [clients, setClients] = useState([]);
+  const [sortColumn, setSortColumn] = useState({ path: 'no', order: 'asc' });
+  const [currentPage, setCurrentPage] = useState(1);
+  const [itemsCount, setItemsCount] = useState(0);
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const getClients = async () => {
+    try {
+      const response = await GetClients(currentPage, defaultPageSize);
+      setItemsCount(response.pagy.count);
+      setClients(response.data);
+    } catch (error) {
+      setItemsCount(0);
+      setClients([]);
+    }
+
+    // let data = mockData.sort((a, b) => {
```

Comment:
@sherviniv no big deal, but could you please delete this commented out code, thanks!